app/assets/javascripts/app/views/publisher/mention_view.js

```diff
@@ -0,0 +1,328 @@
+//= require ../search_base_view
+
+/*
+ * This file is based on jQuery.mentionsInput by Kenneth Auchenberg
+ * licensed under MIT License - http://www.opensource.org/licenses/mit-license.php
+ * Website: https://podio.github.io/jquery-mentions-input/
+ */
+
+app.views.PublisherMention = app.views.SearchBase.extend({
+  KEYS: {
+    PASTE: 118, BACKSPACE: 8, TAB: 9, RETURN: 13, ESC: 27, LEFT: 37,
+    UP: 38, RIGHT: 39, DOWN: 40, COMMA: 188, SPACE: 32, HOME: 36, END: 35
+  },
+
+  settings: {
+    triggerChar: "@",
+    minChars: 2,
+    templates: {
+      wrapper: _.template("<div class='mentions-input-box'></div>"),
+      mentionsOverlay: _.template("<div class='mentions-box'><div class='mentions'><div></div></div></div>"),
+      mentionItemSyntax: _.template("@{<%= name %> ; <%= handle %>}"),
+      mentionItemHighlight: _.template("<strong><span><%= name %></span></strong>")
+    }
+  },
+
+  events: {
+    "keydown #status_message_fake_text": "onInputBoxKeyDown",
+    "keypress #status_message_fake_text": "onInputBoxKeyPress",
+    "input #status_message_fake_text": "onInputBoxInput",
+    "click #status_message_fake_text": "onInputBoxClick",
+    "blur #status_message_fake_text": "onInputBoxBlur",
+    "paste #status_message_fake_text": "onInputBoxPaste"
+  },
+
+  initialize: function(){
+    this.mentionsCollection = [];
+    this.inputBuffer = [];
+    this.currentDataQuery = "";
+    this.mentionChar = "\u200B";
+
+    this.elmInputBox = this.$el.find("#status_message_fake_text");
+    this.elmInputWrapper = this.elmInputBox.parent();
+    this.elmWrapperBox = $(this.settings.templates.wrapper());
+    this.elmInputBox.wrapAll(this.elmWrapperBox);
+    this.elmWrapperBox = this.elmInputWrapper.find("> div").first();
+    this.elmMentionsOverlay = $(this.settings.templates.mentionsOverlay());
+    this.elmMentionsOverlay.prependTo(this.elmWrapperBox);
```

Comment:
It looks like you never use `this.elmWrapperBox` outside of this function and inside of this function you assign one value, use it, assign another one and that use that. In the first case I think you can just remove the variable and use the value instead. In the second case a variable might increase the readability but at least use a local variable.